src/shared/ui/common/regionSubmenu.ts

```diff
@@ -27,6 +27,7 @@ export class RegionSubmenu<T> extends Prompter<RegionSubmenuResponse<T>> {
         private readonly itemsProvider: (region: string) => ItemLoadTypes<T>,
         private readonly dataOptions?: ExtendedQuickPickOptions<T>,
         private readonly regionOptions?: ExtendedQuickPickOptions<T>,
+        private readonly separatorLabel: string = "Selections",
```

Comment:
Maybe "Items" is more common?

```suggestion
        private readonly separatorLabel: string = "Items",
```